templates/base.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+    {% block title %}
+    <title>Taxi Service</title>
+    {% endblock %}
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-sm-3">
+                {% block sidebar %}
+                    {% include "includes/sidebar.html" %}
+                {% endblock %}
+            </div>
+            <div class="col-sm-9">
+                {% block content %}
+                {% endblock %}
+            </div>
+        </div>
+    </div>
+
+</body>
+</html>
```

Comment:
In the requirements, you need to change the indents for html in the PyCharm settings.

Reply:
It's not mentioned in this task, but spoilers: https://github.com/mate-academy/py-task-guideline/blob/main/html_settings/README.MD

Reply:
why didn't you fix it?
and @VadymShkarbul  why did you approve?

Reply:
@Y-Havryliv hello! Because there is no such condition in the readme for this task. And I decided if this is not nessesarly, it is not a reason for rejection. Sorry